app/assets/stylesheets/main.scss

```diff
@@ -5,3 +5,13 @@ $govuk-fonts-path: "/";
 $govuk-images-path: "/";
 
 @import "govuk-frontend/dist/govuk/all";
+
+button.govuk-header__link {
+  background: none !important;
```

Comment:
Can these important-ses be avoided by making the selector more specific?

Reply:
Amended to use govuk sass vars and added the correct focus colour